front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/Selection/Settings/AdminPublicationSearchInput.tsx

```diff
@@ -66,7 +65,6 @@ const AdminPublicationSearchInput = ({
         value={null}
         inputValue={visibleSearchTerm}
         placeholder={''}
-        isDisabled={!adminPublicationIds || isFetching}
```

Comment:
- It seems the bug came from isFetching. So sometimes `isFetching` turns to `true`, which disables the select (which looks like a reset). I was able to reproduce it consistently and with this code removed, it doesn't happen anymore.
- (`adminPublicationIds` is always defined (an empty `[]` rather than `undefined` at first, it seems, so `!adminPublicationIds` is always `false` and an unnecessary condition).)